src/java/com/puppetlabs/puppetserver/JRubyPuppet.java

```diff
@@ -15,7 +15,6 @@
  *
  */
 public interface JRubyPuppet {
-    Map<String, Map> getClassInfoForAllEnvironments();
```

Comment:
We don't have plans for a "get all environments" version of the "environment_classes" endpoint at this point.  I thought it would be better to just purge this method - and the corresponding implementation in `master.rb`.